bin/check_components_diff.rb

```diff
@@ -3,11 +3,54 @@
 
 require_relative '../lib/release_manager'
 require_relative '../lib/components_diff/components_diff'
+require 'terminal-table'
+require 'colorize'
 
-ReleaseManager::ComponentsDiff::Runner.run(ARGV[0] || 'master').each do |component, details|
-  puts "\n"
-  puts "#{component} - #{details[:tag]}"
-  details[:commits].each do |commit|
-    puts "  #{commit.sha} - #{commit.message.delete("\n")}"
+class Presenter
+  attr_reader :rows
+
+  def initialize
+    @rows = []
+  end
+
+  def present
+    ReleaseManager::ComponentsDiff::Runner.run(ARGV[0] || 'master').each do |component, details|
+      print_details component, details
+      next if /core|runtime|api/.match?(component)
+
+      flag = details[:commits].any?
```

Comment:
No need to assign to a variable, just use `add_row(component, details[:tag], details[:commits].any?)`